ENGAGEHF/Bluetooth/Views/ConfirmMeasurementButton.swift

```diff
@@ -16,15 +16,12 @@ struct DiscardButton: View {
     
     
     var body: some View {
-        Button(
-            action: {
-                dismiss()
-            },
-            label: {
-                Text("Discard")
-                    .foregroundStyle(viewState == .idle ? Color.red : Color.gray)
-            }
-        )
+        Button(action: {
```

Comment:
I would generally suggest to stick with the usage of named closures, at least that's the feedback I also provided to Nick.